core_impl/src/test/java/com/google/instrumentation/stats/StatsManagerImplTest.java

```diff
@@ -192,16 +199,26 @@ public void testRecordNonExistentMeasurementDescriptor() {
     assertThat(view.getDistributionAggregations()).hasSize(0);
   }
 
-  // TODO(songya): update this test once we determine how to handle tags that aren't an exact match
-  // for the view.
+
   @Test
   public void testRecordNonExistentTag() {
```

Comment:
This test's name seems less accurate now.  How about something like testRecord_TagNotMatchingView?